src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitStorage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void addFruit(String fruit, int amount) {
+        FruitStorage.fruitRepository.put(fruit, amount);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return FruitStorage.fruitRepository.get(fruit);

```

Comment:
Directly accessing the `fruitRepository` map from another class breaks encapsulation. Consider adding a method in the `FruitStorage` class to handle the retrieval of fruit amounts.